.github/workflows/buildpack-deps.yml

```
@@ -37,6 +37,8 @@ jobs:
 
       - name: comment PR
         if: "env.DOCKER_IMAGE"
-        uses: unsplash/comment-on-pr@b5610c6125a7197eaec80072ea35ef53e1fc6035 #v1.3.1
+        uses: unsplash/comment-on-pr@ffe8f97ccc63ce12c3c23c6885b169db67958d3b #v1.3.0
```

Would be best to have a TODO comment to update this and some explanation on why we can't do it right now.

---

In `v1.3.1` they do a `require_relative` that seem not to work `/entrypoint.sh:5:in `require_relative': cannot load such file -- /lib/github (LoadError)` - so I just went one version down. Also here I did not do any further investigations, because the problem seem to be solved. 

Ah, the reason for this was that I had forked `unsplash/comment-on-pr` in the past, but that version is right now not working anymore, that's why we just changed to the original `unsplash` repository here https://github.com/ethereum/solidity/pull/13394. But this version was not working - so I downgraded that version.

---

We merged https://github.com/ethereum/solidity/pull/13394 too fast without testing - testing was then later done here https://github.com/ethereum/solidity/pull/13396 - next time I will test the stuff already in the same PR.

---

ok, I'd still add at least a short comment about that:
```suggestion
        # NOTE: Can't update to v1.3.1 due to an error: `/entrypoint.sh:5:in 'require_relative': cannot load such file -- /lib/github (LoadError)`
        uses: unsplash/comment-on-pr@ffe8f97ccc63ce12c3c23c6885b169db67958d3b # v1.3.0
```

---

Ok! I added the comment.